tests/network/localnet/test_ovs_bridge.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from libs.net.traffic_generator import is_tcp_connection
+from utilities.virt import migrate_vm_and_verify
+
+
+@pytest.mark.ipv4
+@pytest.mark.usefixtures("nncp_localnet_on_secondary_node_nic")
+@pytest.mark.polarion("CNV-11905")
+def test_connectivity_over_migration_between_ovs_bridge_localnet_vms(
```

Comment:
test name is a bit confusing as `migrate_vm_and_verify` waits for migration to end. connectivity check is done after the migration and not during

Reply:
Connectivity is done over migration - the connection is initiated before the migration, and it should stay up until after the migration.
We will have another scenario to test new connections after migration, but this is not the current scenario.

Reply:
Connectivity is checked during migration in the sense it is checked before it starts [1] and confirmed it did not break during the migration (i.e. the connection is still there).

[1] Fixture `localnet_ovs_bridge_client` does a `assert client.is_running()`.

Reply:
should have been igrate_vm_and_verify(vm=localnet_ovs_bridge_client.vm, wait_for_migration_success=False) but will not block the pr

Reply:
Why `wait_for_migration_success=False` ?
We need the migration to finish and see that the connection has survived during that process.
To be clear, if the connection breaks during the migration, it will not auto-recover and we will see it when the migration finishes.

Reply:
Since is was approved by both Edi and Ruth after this conversation ended, I'm resolving this comment.